distribution/changelog.txt

```diff
@@ -12,6 +12,7 @@
 - Improved: [#22352] The object selection window now groups relevant object tabs together.
 - Improved: [#22357] Error messages are now themeable and easier to read.
 - Improved: [#22361] Add additional colour preset for the Observation Tower.
+- Change: [#12292] The 'Toggle visibility of toolbars' shortcut is no longer assigned by default.
```

Comment:
Could you replace these typewriter quotes?

(Note: we should probably amend the CI to check for this.)

Reply:
```suggestion
- Change: [#12292] The ‘Toggle visibility of toolbars’ shortcut is no longer assigned by default.
```